docs/01-introduction.Rmd

```diff
@@ -183,7 +183,19 @@ Note that [R will silently ignore the last line of your `.Rprofile`](https://yih
 
 ## R Markdown vs. Markdown {#output-format}
 
-If you are not familiar with R Markdown\index{R Markdown}, please see Appendix \@ref(r-markdown) for a quick tutorial. When you create a new post, you have to decide whether you want to use R Markdown or plain Markdown\index{Markdown}, as you can see from Figure \@ref(fig:new-post). The main differences are:
+If you are not familiar with R Markdown\index{R Markdown}, please see Appendix \@ref(r-markdown) for a quick tutorial. When you create a new post, you have to decide whether you want to use R Markdown or plain Markdown\index{Markdown}, as you can see from Figure \@ref(fig:new-post). 
+
+The following table summarises the differences between the three options, followed by more detailed explanations below.
+
+|Features      |.Rmd |.Rmarkdown |.md |
+|:------------|:---|:---------|:--|
+|bibliography |yes |no        |no |
+|run r code         |yes  |yes        |no |
+|task list         |no  |yes        |yes |
+|mathjax         |yes  |no        |yes |
```

Comment:
The support for MathJax is the same in `.md` and `.Rmarkdown` (there is no native support, and you have to use tricks). See line 246 below.